packages/Rakefile

```
@@ -20,39 +20,22 @@ namespace :version do
   end
 end
 
-namespace :ubuntu do
-  desc "Upload ubuntu packages"
-  task :upload do
-    packages.each do |package|
-      cd(package) do
-        ruby("-S", "rake", "ubuntu")
-      end
+desc "Release ubuntu packages"
+task :ubuntu do
+  packages.each do |package|
+    cd(package) do
+      ruby("-S", "rake", "ubuntu")
     end
   end
 end
 
-desc "Release ubuntu repositories"
-ubuntu_tasks = [
-  "ubuntu:upload",
-]
-task :ubunut => ubuntu_tasks
-
-
 [:apt, :yum].each do |target_namespace|
```

```suggestion
[:apt, :yum].each do |target|
```

Because we don't use it for namespace anymore.